airflow-core/src/airflow/ui/src/i18n/locales/fr/common.json

```diff
@@ -0,0 +1,271 @@
+{
+  "admin": {
+    "Config": "Configuration",
+    "Connections": "Connexions",
+    "Plugins": "Plugins",
+    "Pools": "Pools",
+    "Providers": "Providers",
+    "Variables": "Variables"
+  },
+  "asset_one": "Asset",
+  "asset_other": "Assets",
+  "assetEvent_one": "Événement d'Asset",
+  "assetEvent_other": "Événements d'Asset",
+  "backfill_one": "Rattrapage",
+  "backfill_other": "Rattrapages",
+  "browse": {
+    "auditLog": "Journal d'audit",
+    "xcoms": "XComs"
+  },
+  "createdAssetEvent_one": "Événement d'Asset créé",
+  "createdAssetEvent_other": "Événements d'Asset créés",
+  "dag_one": "Dag",
+  "dag_other": "Dags",
+  "dagDetails": {
+    "catchup": "Rattrapage",
+    "concurrency": "Concurrence",
+    "dagRunTimeout": "Délai d'exécution du Dag",
+    "defaultArgs": "Arguments par défaut",
+    "description": "Description",
+    "documentation": "Documentation du Dag",
+    "fileLocation": "Emplacement du fichier",
+    "hasTaskConcurrencyLimits": "Limites de concurrence par tâche",
+    "lastExpired": "Date d'experiration",
+    "lastParsed": "Dernière analyse",
+    "latestDagVersion": "Dernière version du Dag",
+    "latestRun": "Dernière exécution",
+    "maxActiveRuns": "Exécutions actives max.",
+    "maxActiveTasks": "Tâches actives max.",
+    "maxConsecutiveFailedDagRuns": "Échecs consécutifs max.",
+    "nextRun": "Prochaine exécution",
+    "owner": "Propriétaire",
+    "params": "Paramètres",
+    "schedule": "Planification",
+    "tags": "Tags"
+  },
+  "dagId": "ID du Dag",
+  "dagRun": {
+    "conf": "Configuration",
+    "dagVersions": "Version(s) du Dag",
+    "dataIntervalEnd": "Fin de l'intervalle de données",
+    "dataIntervalStart": "Début de l'intervalle de données",
+    "lastSchedulingDecision": "Dernière décision de planification",
+    "queuedAt": "Mis en file à",
+    "runAfter": "Exécuté après",
+    "runType": "Type d'exécution",
+    "sourceAssetEvent": "Événement d'Asset source",
+    "triggeredBy": "Déclenché par"
+  },
+  "dagRun_one": "Exécution de Dag",
+  "dagRun_other": "Exécutions de Dag",
+  "dagWarnings": "Avertissements/erreurs du Dag",
+  "defaultToGraphView": "Vue par défaut : graphe",
+  "defaultToGridView": "Vue par défaut : grille",
+  "direction": "Direction",
+  "docs": {
+    "documentation": "Documentation",
+    "githubRepo": "Repo GitHub",
+    "restApiReference": "Référence API REST"
+  },
+  "duration": "Durée",
+  "endDate": "Date de fin",
+  "error": {
+    "back": "Retour",
+    "defaultMessage": "Une erreur inattendue est survenue",
+    "home": "Accueil",
+    "notFound": "Page introuvable",
+    "title": "Erreur"
+  },
+  "expression": {
+    "all": "TOUS",
+    "and": "ET",
+    "any": "N'importe lequel",
+    "or": "OU"
+  },
```

Comment:
Since we cannot use uppercase for the translation of `any`, I prefer not to use it for the other translations. (to be consolidated with `expression` block)